core/scheduler/build.gradle

```diff
@@ -52,6 +52,16 @@ buildscript {
     }
 }
 
+protobuf {
+    protoc {
+        if (osdetector.os == "osx") {
+            artifact = 'com.google.protobuf:protoc:3.4.0:osx-x86_64'
```

Comment:
IIRC, we cant' upgrade the protoc version for now because some libraries are dependent on this version.
And protoc-3.4.0 does not support arm architecture yet, so I chose this way.
This also requires rosetta installed.

https://github.com/grpc/grpc-java/issues/7690#issuecomment-760332746